student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age:, classroom:)
```

Comment:
- [ ] Please add `name` and `parent permission` as parameters to the initialize method. It can look something similar to the line of code below.
```
def initialize(age, classroom, name = "Unknown", parent_permission = true)
```